build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -153,10 +153,15 @@ public void CrossgenDirectory(BuildTargetContext c, string pathToAssemblies)
 
                 IList<string> crossgenArgs = new List<string> {
                     "-readytorun", "-in", file, "-out", tempPathName,
-                    "-JITPath", GetLibCLRJitPathForVersion(),
                     "-platform_assemblies_paths", platformAssembliesPaths
                 };
 
+                if (CurrentPlatform.IsUnix)
+                {
+                    crossgenArgs.Add("-JITPath");
```

Comment:
Correct. The Windows version of crossgen comes from the closed build which statically links the JIT. When you switch to the current packages that will be published from the open, then Windows would also need to specify this argument.

CC @wtgodbe who has the packages published from the master branch in the open to test feed and will soon enable it for RTM branch packages.